lib/twitter_web/templates/user/new.html.eex

```diff
@@ -0,0 +1,32 @@
+<h1>New user</h1>
+
+<%= form_for @changeset, Routes.user_path(@conn, :create), fn f -> %>
+<%= if @changeset.action do %>
+  <div class="alert alert-danger">
+    <p>Ooops, something is wrong</p>
+  </div>
+<% end %>
+
+    <div>
```

Comment:
try using some formatter for .eex files, so that the identation is consistent, I think you can download an extension for vs code if you search .eex